### PR TITLE
New Question related to Creating Packages (Python)

### PR DESCRIPTION
**Pull request by thegeekyb0y**
Checklist:

- [x] I have read [freeCodeCamp's contribution guidelines](https://contribute.freecodecamp.org).
- [x] My pull request has a [descriptive title](https://contribute.freecodecamp.org/#/how-to-open-a-pull-request?id=prepare-a-good-pr-title) (**not** a vague title like `Update index.md`)

Closes #XXXXX

Added a question related to Creating Packages in python, putted 3 distractors, a proper explanation and a link to freecodecamp's blog.

### CODE REVIEW

File: src/data/python-quiz.ts [hunk 1]
```diff
@@ -928,6 +928,15 @@ const pythonQuiz = [
     Explanation: "Python is an interpreted programming language, which supports object-oriented, structured, and functional programming.", 
     Link: "https://en.wikipedia.org/wiki/Python_(programming_language)", 
   },
+  {
+    Question: "In Python, which file is compulsory to create a package?",
+    Answer: "__init__",
+    Distractor1: "__main__",
+    Distractor2: "main.py",
+    Distractor3: "__package__",
+    Explanation: "We cannot create a python package without adding a __init__.py file, This file will tell Python to treat directories as modules.",
```
Comment (jdwilkin4) [inline src/data/python-quiz.ts:937]:
```suggestion
    Explanation: "We cannot create a Python package without adding a __init__.py file, This file will tell Python to treat directories as modules.",
```

**Review by jdwilkin4 — CHANGES_REQUESTED** [review]
Thank you for your contribution!

Please review the requested changes.
Once those changes have been implemented, then we can merge your PR into main. 👍

**Review by jdwilkin4 — APPROVED** [review]
Congrats on another contribution to our repository. 🎉
We look forward to future contributions. 👍

**Comment by thegeekyb0y** [comment]
Can I put more questions ?

**Comment by jdwilkin4** [comment]
@thegeekyb0y 
You can create a new PR for new questions since this has already been reviewed and merged 👍

**Pull request merged**